src/components/options-menu/options-menu.tsx

```diff
@@ -0,0 +1,208 @@
+import React, { ChangeEvent, useCallback, useMemo, useRef, useState } from "react";
+import { AnalogStickSize, AppOptions, ColorOption, UrlOptions } from "../../services/url-options.service";
+import './options-menu.scss';
+
+const lettersOnly: RegExp = /^[a-zA-Z]/;
+
+export default function OptionsMenu(props: OptionsMenuProps): React.ReactElement {
+
+  const [formOptions, setFormOptions] = useState<OptionsMenuState>({
+    wheelImgUrl: {
+      value: props.appOptions.wheelImgUrl
+    },
+    wheelColor: {
+      chooser: (Object.values(ColorOption) as string[]).includes(props.appOptions.wheelColor) ? "predefined" : 'custom',
```

Comment:
could you use constants or enums instead of repeating those strings? "predefined" and "custom"

Reply:
Done: 92b39cdb050